### DISCUSSION TITLE
How to change the URL of the "admin" dashboard

### DISCUSSION THREAD

**Discussion by rick427** [avitorio/outstatic#97] [Q&A] [answered]
Hi, team **Outstatic**. First off I want to congratulate you guys for building a solid product, I've been looking for a good open-source CMS for a static project on Next.js but unfortunately haven't seen one quite like yours.

A few things though, after following the steps listed in the docs, I noticed that the default URL to access the CMS dashboard is "**{{BASE_URL}}/outstatic**" can I change this to "**{{BASE_URL}}/admin**" if so how can it be done ? I tried it earlier but I got a 404 error on the client maybe am missing something.

Thanks :)

**Comment by avitorio** [comment, marked as answer]
Hi @rick427, thanks for the kind words. I hope you are enjoying the product.

Unfortunately there's no easy way to change the dashboard url to `/admin` for the moment as the paths are hardcoded. We are working on some new features but this will probably come at some point. All the best!
